app/views/api/umgap/index.html.erb

```diff
@@ -0,0 +1,144 @@
+<div class="apidocs row">
+  <div class="col-xs-9">
+    <h1 class='api-title'>
+      <span class="initialism">UMGAP</span> documentation
+    </h1>
+    <p class='lead'>
+      Use the Unipept Metagenomics Analysis Pipeline (<span class='initialism'>UMGAP</span>) to analyse and present your findings using frequency tables and interactive visualizations. 
```

Comment:
The introduction lacks the specification of which data this is about. The lead here doesn't even mention data. The next one only talks about metagenomics analyses. (short) shotgun metagenomics reads should at least be mentioned somewhere.